spec/shr_oncology.txt

```diff
@@ -117,52 +384,55 @@ Description:	"The stage of a breast cancer. Different staging systems use differ
 				ObservationContext.ResultValue should be from http://loinc.org/vs/LL240-3
 
 
-EntryElement:	DCISNuclearGrade
-Concept:		MTH#C18513  // What is the proper LOINC code?
+EntryElement:	NuclearGrade
```

Comment:
After group discussion, revert references to NuclearGrade back to DCISNuclearGrade.